ffi/dotnet/IronRdp.sln

```diff
@@ -30,5 +31,16 @@ Global
 		{6BB8B946-60BA-4DB8-B3DB-EF9CB6CF8A14}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{6BB8B946-60BA-4DB8-B3DB-EF9CB6CF8A14}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{6BB8B946-60BA-4DB8-B3DB-EF9CB6CF8A14}.Release|Any CPU.Build.0 = Release|Any CPU
+		{C918DB8D-5797-48EE-8813-CD3906DCE253}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Temporarily added NowProto projects to `ffi` solution, but not yet decided where C# NowProto implementation should reside. 
- Alongside IronRDP - The same repo as with rust implementation seems convenient (we could even make some integration tests for server <-> client protocol consistency).
- However, on the other hand, IronRDP repo once again will become even more bloated with loosely related code. Should we separate this into its own repo maybe? (alongside with rust's `now-proto-pdu`. Btw, only need ironrdp-core crate from IronRDP, so it should be easily detachable from repo)

Reply:
Indeed, at this point, we could create a new repository for NowProto. All IronRDP crates are published on crates.io, so it’s now easy to reuse any of them anywhere, including ironrdp-server.